app/src/pages/Lobby/Inputs/index.tsx

```diff
@@ -111,13 +126,19 @@ export function SecondsPerTurnInput(props: {
   return (
     <TextField
       type="number"
-      label="Seconds Per Turn"
+      label={t("settings.turns.secondsPerTurn.label", "Seconds Per Turn")}
       variant="outlined"
       size="medium"
       required
-      helperText={<HelperText>Usually between 30 or 60</HelperText>}
+      helperText={
+        <HelperText>
+          {t(
+            "settings.turns.secondsPerTurn.helper",
+            "Usually between 30 or 60"
```

Comment:
do numbers get translated? or would we do something like

```
          {t(
            "settings.turns.secondsPerTurn.helper",
            "Usually between {{ numberThirty }} or {{ numberSixty }}",
            { numberSixty: 60, numberThirty: 30 }
          )}
```

Reply:
In general, numbers are extracted out into a `count` variable so you can do pluralization – which we do in a few contexts.

I could go either way for helper text like this. You're probably right – the more correct way to do this would involve pulling out the numbers so we could change them without having to update language strings, so I'll probably change this to something like:

```ts
{t(
  "settings.turns.secondsPerTurn.helper",
  "Usually between {{ secondsLowerBound }} or {{ secondsUpperBound }}",
  {
    secondsLowerBound: 30,
    secondsUpperBound: 60,
  }
)}
```

Reply:
Updated in 957d64b761a54c0d1310f2dc6b308f416e6e1642.